contributing.rst

```diff
@@ -1,3 +1,15 @@
+How to install for developing
+-----------------------------
+
+Clone the repository with::
+
+    git clone git@github.com:PLOS/allofplos.git
```

Comment:
Is there a reason you prefer this over `https://github.com/PLOS/allofplos.git`?

Reply:
I used this because it is what is displayed in the GitHub UI, but the https option is better since works in more cases. Will change it.